plugins/inputs/mongodb/mongostat.go

```diff
@@ -681,8 +692,19 @@ func NewStatLine(oldMongo, newMongo MongoStatus, key string, all bool, sampleSec
 		returnVal.BytesReadInto = newStat.WiredTiger.Cache.BytesReadInto
 		returnVal.PagesEvictedByAppThread = newStat.WiredTiger.Cache.PagesEvictedByAppThread
 		returnVal.PagesQueuedForEviction = newStat.WiredTiger.Cache.PagesQueuedForEviction
+		returnVal.PagesReadIntoCache = newStat.WiredTiger.Cache.PagesReadIntoCache
+		returnVal.PagesRequestedFromCache = newStat.WiredTiger.Cache.PagesRequestedFromCache
 		returnVal.ServerEvictingPages = newStat.WiredTiger.Cache.ServerEvictingPages
 		returnVal.WorkerThreadEvictingPages = newStat.WiredTiger.Cache.WorkerThreadEvictingPages
+
+		returnVal.InternalPagesEvicted = newStat.WiredTiger.Cache.InternalPagesEvicted
+		returnVal.ModifiedPagesEvicted = newStat.WiredTiger.Cache.ModifiedPagesEvicted
+		returnVal.UnmodifiedPagesEvicted = newStat.WiredTiger.Cache.UnmodifiedPagesEvicted
+
+		returnVal.FlushesTotalTime = float64(newStat.WiredTiger.Transaction.TransCheckpointsTotalTimeMsecs) / float64(time.Millisecond)
```

Comment:
Our latest style for durations is to report as an integer in nanoseconds: TotalTimeMsecs * time.Millisecond

Reply:
I can't find any exists metrics in nanoseconds.
In `inputs/cpu/cpu.go` time reported as `float64` in `seconds`.
I don't known correct type for `FlushesToTotalTime`: `float64`, `int64`, `time.Duration`, `internal.Duration` or something else...

Reply:
The time and duration values are not nearly as uniform as I would like, though I can think of several that are using ns currently (zipkin, internal, kapacitor).  In the future I would like to normalize these time datatypes more, perhaps even adding them as first class types in Telegraf.

Use int64 for the time in ns.